foundations/foundation-auth/src/main/java/org/apache/servicecomb/foundation/auth/rbac/Cipher.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
```

Comment:
This class and implementation can move to foundation-common, do not need module foundation-auth

Reply:
has removed to foundation-common